src/freenet/node/NewPacketFormat.java

```diff
@@ -973,9 +965,9 @@ public void checkForLostPackets() {
 		// LOCKING: No packet may be sent while connected = false.
 		// So we guarantee that no more packets are sent by setting this here.
 		synchronized(sendBufferLock) {
-			for(HashMap<Integer, MessageWrapper> queue : startedByPrio) {
+			for(Map<Integer, MessageWrapper> queue : startedByPrio) {
 				if(items == null)
```

Comment:
Is this `if` seriously only here because someone didn’t want to initialize the `items` where they were declared?